elements/components/cups-base.bst

```diff
@@ -1,12 +1,11 @@
-# utils/update-elements.sh: Derived from freedesktop-sdk.bst freedesktop-sdk-24.08.23 components/cups-base.bst
+# utils/update-elements.sh: Derived from freedesktop-sdk.bst freedesktop-sdk-24.08.25 components/cups-base.bst
 
 kind: autotools
 description:
   Printing system.
 
-  This element overrides its equivalent in Freedesktop SDK so we can
-  name the system group "sudo" instead of "wheel", for backwards
-  compatibility with EOS6.
+  Overridden to rename the 'wheel' groupe to 'sudo', for backwards compatibility
+  with Endless OS 6.
```

Comment:
That comment was actually incomplete -- it was also to add some sysuser ids as seen in `files/cups`.

You can link to doc/overview/sysusers.md for a writeup of this

typo 'groupe'